### PR TITLE
#47 Estimated correlated drift by default

### PR DESCRIPTION
**Pull request by ScottClaessens**
These changes make it the default to estimate correlated drift.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -113,7 +113,7 @@ in the optimal trait value of one variable which results from a one median
 absolute deviation increase in another variable.
 
 ```{r authority-delta-theta, fig.height=5, fig.width=5}
-coev_plot_delta_theta(fit)
+coev_plot_delta_theta(fit, limits = c(-5, 50))
```
Comment (ErikRingen) [inline README.Rmd:116]: Why such a large positive limit? To show that they keep increasing even at large values? Is this now far beyond the sample range of the latent trait?

**Comment by ScottClaessens** [comment]
@ErikRingen let me know if you have any thoughts on my comments above, before I merge this branch :)

File: R/coev_plot_selection_gradient.R [hunk 2]
```diff
@@ -146,7 +146,7 @@ coev_plot_selection_gradient <- function(object, var1, var2,
     # scale by mad for response variable
     out <- out / mads[resp_id]
     # divide by sigma scaled by mad for response variable
-    sigma <- Q_diag[resp_id] / mads[resp_id]
+    sigma <- Q_sigma[resp_id] / mads[resp_id]
```
[the remark this answers is not in the document]
Comment (ScottClaessens) [reply R/coev_plot_selection_gradient.R:149]: Or actually, should it be `Q_sigma[resp_id]^2`

[the remark this answers is not in the document]
Comment (ErikRingen) [reply R/coev_plot_selection_gradient.R:149]: Yes it should be sqrt now given changes to Stan code

[the remark this answers is not in the document]
Comment (ScottClaessens) [reply R/coev_plot_selection_gradient.R:149]: But `Q_sigma` is now the vector of standard deviations, whereas the diagonal of the `Q` matrix is now the variances. That's why I thought that it should be `Q_sigma[resp_id]^2` instead, to align with the Q matrix.

[the remark this answers is not in the document]
Comment (ErikRingen) [reply R/coev_plot_selection_gradient.R:149]: Ah right, I was thrown off by the Q labe. So yeah as you say or you could have `sqrt(Q[resp_id, resp_id])`.

[the remark this answers is not in the document]
Comment (ScottClaessens) [reply R/coev_plot_selection_gradient.R:149]: I'll make this change.

**Pull request merged**